src/Standards/Squiz/Sniffs/Operators/ComparisonOperatorUsageSniff.php

```diff
@@ -157,7 +157,7 @@ public function process(File $phpcsFile, $stackPtr)
 
         for ($i = $start; $i <= $end; $i++) {
             $type = $tokens[$i]['code'];
-            if (in_array($type, array_keys(self::$invalidOps[$tokenizer])) === true) {
+            if (array_key_exists($type, self::$invalidOps[$tokenizer]) === true) {
```

Comment:
Using `isset()` would probably be more stable, as `array_key_exists()` will still return `true`, even when the key has no value.